05week/checkers.js

```diff
@@ -80,47 +80,112 @@ function Board() {
     console.log(string);
   };
 
-  // Your code here
 //create an array for checkers
-//create checkers on board
-this.checkers = [];
+//creates checkers on board
+  this.checkers = [];
 //creates white checkers on board, adds to checkers array
   this.createCheckers = function() {
     for (let i = 0; i <= 2; i++) {
-        for (let j = 1; j <= 7; j+=2) {
-          if ((i === 1) && (j % 2 !== 0)) (j = j - 1);
-          let whiteChecker = new Checker('white');
-          this.grid[i][j] = whiteChecker;
-          this.checkers.push(whiteChecker);
+      for (let j = 1; j <= 7; j+=2) {
+        if ((i === 1) && (j % 2 !== 0)) (j = j - 1);
+        let whiteChecker = new Checker('white');
+        this.grid[i][j] = whiteChecker;
+        this.checkers.push(whiteChecker);
+      }
+    }
+//creates black checkers on board, adds to checkers array
+    for (let i = 5; i <= 7; i++) {
+      for (let j = 1; j <= 7; j+=2) {
+        if ((i === 5 || i === 7) && (j % 2 !== 0)) (j = j - 1);
+        let blackChecker = new Checker('black');
+        this.grid[i][j] = blackChecker;
+        this.checkers.push(blackChecker);
+      }
+    };
   }
+  this.selectChecker = function(position) {
+    console.log(this.grid);
+  };
+}
```

Comment:
Move this `}` on line 109 after your `killChecker` method to properly envelop all your logic in the `Board` function.